opta/core/kubernetes.py

```diff
@@ -222,12 +222,12 @@ def current_image_tag(layer: "Layer",) -> Optional[str]:
     return None
 
 
-def create_namespace(layer_name: str) -> None:
+def create_namespace_if_not_exists(layer_name: str) -> None:
     load_kube_config()
     v1 = CoreV1Api()
     namespaces = v1.list_namespace(field_selector=f"metadata.name={layer_name}")
     if len(namespaces.items) == 0:
```

Comment:
Do we need this check if we're using `v1.create_namespace_if_not_exists`?

Reply:
yes? K8s returns a 409 error otherwise and this seemed cleaner than handling the 409

Reply:
ah wondering if the 409 will lead to an exception on line 228?